core/services/ocrcommon/transmitter.go

```diff
@@ -203,3 +218,57 @@ func (t *ocr2FeedsTransmitter) forwarderAddress(ctx context.Context, eoa, ocr2Ag
 
 	return forwarderAddress, nil
 }
+
+type ocr2FeedsDualTransmission struct {
+	transmitter ocr2FeedsTransmitter
+
+	secondaryContractAddress common.Address
+	secondaryFromAddress     common.Address
+	secondaryMeta            map[string][]string
+}
+
+func (t *ocr2FeedsDualTransmission) CreateEthTransaction(ctx context.Context, toAddress common.Address, payload []byte, txMeta *txmgr.TxMeta) error {
+	// Primary transmission
+	errPrimary := t.transmitter.CreateEthTransaction(ctx, toAddress, payload, txMeta)
```

Comment:
would it be possible to update CreateEthTransaction interface so that it can also be used below ? (instead of calling the underlying txm directly)